test/gtest/ucs/test_log.cc

```diff
@@ -17,8 +17,8 @@ class log_test : private ucs::clear_dontcopy_regions, public ucs::test {
 public:
     virtual void init() {
 #ifdef __SANITIZE_ADDRESS__
-        /* BUG: on CentOS8 if LD_PRELOAD=libasan.so
-         * grep returns 1 return code even if pattern was found
+        /* BUG: on Ubuntu22.04 if LD_PRELOAD=libasan.so
```

Comment:
why change back to Ubuntu?

Reply:
Because it failing on Ubuntu if `LD_PRELOAD=libasan` is set